reference/auth/models/palmtreeresponse.v1.yaml

```diff
@@ -0,0 +1,24 @@
+title: Palm Tree Response
+description: Data retrieved from Palm Tree's API.
+type: object
+properties:
+  type:
+    type: string
+  enrollment:
+    type: object
+    properties:
+      body:
+        type: string
+        description: PEM encoded certificate.
```

Comment:
In another comment, I mentioned a difference between "base64 encoded", "base64 encoded DER" and now it gets even weirder as we add "PEM" to mix.

So I urge you be consistent and exact in these terms to save people headaches later.